lib/openhab/dsl/items/group_item.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+require 'forwardable'
+require 'java'
+require 'openhab/dsl/items/item_command'
+require 'openhab/dsl/items/item_delegate'
+require 'openhab/core/entity_lookup'
+
+module OpenHAB
+  module DSL
+    module Items
+      #
+      # Delegator to OpenHAB Group Item
+      #
+      class GroupItem
+        extend OpenHAB::DSL::Items::ItemCommand
+        extend OpenHAB::DSL::Items::ItemDelegate
+        include Enumerable
+
+        def_item_delegator :@group_item
+
+        #
+        # Create a new GroupItem
+        #
+        # @param [Java::Org::openhab::core::items::GroupItem] group_item OpenHAB GroupItem to delegate to
+        #
+        def initialize(group_item)
+          @group_item = group_item
+
+          item_missing_delegate { @group_item }
+          add_state_methods
```

Comment:
item_missing_delegate { OpenHAB::Core::EntityLookup.decorate_items(@group_item.members.to_a) }

Reply:
Wouldn't that mess with things like `Group.state` and other methods delegated to the OH GroupItem?